content/doc/index.adoc

```diff
@@ -4,26 +4,45 @@ title: Jenkins Documentation
 section: doc
 ---
 
+== What is Jenkins?
+
 Jenkins is a self-contained, open source automation server which can be used to
 automate all sorts of tasks such as building, testing, and deploying software.
+
 Jenkins can be installed through native system packages, Docker, or even run
-standalone by any machine with the Java Runtime Environment installed.
+standalone by any machine with the Java Runtime Environment (JRE) installed.
 
 == Guided Tour
 
-This guided tour will use the "standalone" Jenkins distribution which requires
-Java 8. A system with more than 512MB of RAM is also recommended.
+This guided tour introduces you to the basics of using Jenkins and its main
+feature, Jenkins Pipeline. This tour uses the \'standalone' Jenkins distribution,
```

Comment:
Why single quotes?  I'm not opposed per se, I'd like to make sure we're being consistent.

Reply:
Ah! It's because I'm a creature of habit :-) ... I only thought there were spelling differences between Australian/UK English and US English, but apparently that extends a bit to punctuation usage too ([as described in this article about punctuation usage])(https://www.scribendi.com/advice/when_to_use_double_or_single_quotation_marks.en.html).

Anyway, I'm happy to change habits - for consistency, I've changed these back to double quotes.